### DISCUSSION TITLE
C# equivalent to PascalABC.NET's `array of const`?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3039] [Q&A] [answered]
I guess `params string[] args`?

**Comment by ghost** [comment]
@SunSerega

**Reply by SunSerega** [reply, marked as answer]
I saw it, but only now got to my PC.

More importantly, what in the world is an `array of const`?
And what do you think the `params` keyword means? It exists in both Pascal and C# and does the same thing with the same syntax.

**Reply by ghost** [reply]
`array of const` is Delphi syntax. I think PascalABC.NET uses the same syntax as Delphi. I'm sorry.

**Reply by SunSerega** [reply]
Ah, I see. `Phind` says if this is Delphi - `const` inside an array denotes a variadic type, which can accept a few basic types.
Well, both Pascal and C# have the `object` type, which can fit not just a few basic ones, but any non-pointer type (and pointer can be packed into `record` (`struct`), which itself does fit into `object`)
